src/main/java/com/airbnb/billow/AWSDatabase.java

```diff
@@ -79,6 +77,37 @@
         this.ec2SGs = ec2SGbuilder.build();
 
         /*
+         * IAM keys
+         */
+
+        log.info("Getting IAM keys");
+        String accountNumber = ""; // we need this to describe RDS
+        final ImmutableList.Builder<IAMUserWithKeys> usersBuilder = new ImmutableList.Builder<IAMUserWithKeys>();
+
+        final ListUsersRequest listUsersRequest = new ListUsersRequest();
+        ListUsersResult listUsersResult;
+        do {
+            log.debug("Performing IAM request: {}", listUsersRequest);
+            listUsersResult = iamClient.listUsers(listUsersRequest);
+            final List<User> users = listUsersResult.getUsers();
+            log.debug("Found {} users", users.size());
+            for (User user : users) {
+                final ListAccessKeysRequest listAccessKeysRequest = new ListAccessKeysRequest();
+                listAccessKeysRequest.setUserName(user.getUserName());
+                final List<AccessKeyMetadata> accessKeyMetadata = iamClient.listAccessKeys(listAccessKeysRequest).getAccessKeyMetadata();
+
+                final IAMUserWithKeys userWithKeys = new IAMUserWithKeys(user, ImmutableList.<AccessKeyMetadata>copyOf(accessKeyMetadata));
+                usersBuilder.add(userWithKeys);
+
+                if (accountNumber.isEmpty()) {
+                    accountNumber = user.getArn().split(":")[4];
```

Comment:
No. Listing all IAM users is part of billow's feature.

I'm just trying to steal an account number from this loop.